### PR TITLE
sort directories

### PR DESCRIPTION
**Pull request by AndreaAlca**

### CODE REVIEW

File: lib/src/main/kotlin/com/github/bratek20/hla/directory/impl/DirectoriesLogic.kt [hunk 1]
```diff
@@ -11,7 +11,7 @@ class DirectoriesLogic: Directories {
             "File does not exist or not dir for path: $nioPath"
         }
 
-        val allFiles = file.listFiles()
+        val allFiles = file.listFiles()?.sorted()
```
Comment (bratek20) [inline lib/src/main/kotlin/com/github/bratek20/hla/directory/impl/DirectoriesLogic.kt:14]: lets add tests for Directory module showing example of expected sorting

Comment (AndreaAlca) [reply lib/src/main/kotlin/com/github/bratek20/hla/directory/impl/DirectoriesLogic.kt:14]: added

Comment (bratek20) [reply lib/src/main/kotlin/com/github/bratek20/hla/directory/impl/DirectoriesLogic.kt:14]: wrong place

Comment (AndreaAlca) [reply lib/src/main/kotlin/com/github/bratek20/hla/directory/impl/DirectoriesLogic.kt:14]: moved in correct file

**Pull request merged**